app/controllers/api/v1/donation_tokens_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::DonationTokenssController < Api::V1::BaseController
```

Comment:
We don't actually need this. My thinking here was that the client might want to let the user know they've already used the token, but that's probably better done elsewhere.